source/common/common/fancy_logger.h

```diff
@@ -94,7 +94,7 @@ FancyContext& getFancyContext();
     static std::atomic<spdlog::logger*> flogger{0};                                                \
     spdlog::logger* local_flogger = flogger.load(std::memory_order_relaxed);                       \
     if (!local_flogger) {                                                                          \
-      getFancyContext().initFancyLogger(FANCY_KEY, flogger);                                       \
```

Comment:
let's be more explicit, use `::EnvoY::getFancyContext`?

Reply:
done in 41c30f5